tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py

```diff
@@ -1297,14 +1300,15 @@ def default_partition_fn(keys, shard_num):
         parameters by *adding delta* instead of *setting*, which solves the
         race condition problem among workers during backpropagation in
         large-scale distributed asynchronous training.
-
+      short_file_name: If True, the file name will not use scope name as prefix and create_slots will not
+          use op_name to avoid file name over 255. the default is False to keep the same behavior as before.
     Returns:
       A `Variable` object.
     """
-  var_ = None
   scope = variable_scope.get_variable_scope()
   scope_store = variable_scope._get_default_variable_store()
-  full_name = scope.name + "/" + name if scope.name else name
+
+  full_name = scope.name + "/" + name if scope.name and not short_file_name else name
```

Comment:
According to the [TF code](https://github.com/tensorflow/tensorflow/blob/35c9ff922925fc7c605b720fce8f0574be5bd29a/tensorflow/python/ops/variable_scope.py#L822), there is no need to compound the scope.name onto Variable name. So I agree with this commit.